coursebuilder/modules/explorer/_static/css/style.css

```diff
@@ -3594,7 +3594,7 @@ footer .footer-bottom .footer-social-icons a { color: black; }
 
 .course-card { position: relative; }
 
-.course-card img { max-width: 100%; max-height: 400px; }
+.course-card img { max-width: 100%; max-height: 400px; object-fit: contain }
```

Comment:
@Tawakalt please revert the max-height too 100.

Reply:
@chegejames 100px or 100%?

Reply:
@Tawakalt 100%